lib/react.dart

```diff
@@ -1273,19 +1205,8 @@ class SyntheticPointerEvent extends SyntheticEvent {
     this.twist,
     this.pointerType,
     this.isPrimary,
-  ) : super(
-            bubbles,
-            cancelable,
-            currentTarget,
-            defaultPrevented,
-            preventDefault,
-            stopPropagation,
-            eventPhase,
-            isTrusted,
-            nativeEvent,
-            target,
-            timeStamp,
-            type) {}
+  ) : super(bubbles, cancelable, currentTarget, defaultPrevented, preventDefault, stopPropagation, eventPhase,
```

Comment:
you get it..... its not a big deal if there isn't a way, this is only like not great because it has 1000 arguments, which probably doesn't happen tooo often.